content/authors/akitsugu-ito/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Akitsugu Ito
+bio: |
+   I have been working in the industry as a security engineer for about 11 years. 
+   I specialize in CSIRT/PSIRT construction and operation, information security management and incident handling, and product security management.
+   CISSP/SSCP
+avatar: /authors/akitsugu-ito/IMG_0880.JPG
```

Comment:
It's better to rename the file `IMG_0880.JPG` to something clearer like `akitsugo-ito-profile.jpg`. There could especially be an issue with all-caps `JPG` on some browsers, since it's non-standard (DOS-style extension).

Reply:
The image is also huge (3317x3724). Could it be resized and properly compressed for web delivery?